src/widget/woverview.cpp

```diff
@@ -94,6 +93,18 @@ WOverview::WOverview(
             this, &WOverview::onTrackAnalyzerProgress);
 
     connect(m_pCueMenuPopup.get(), &WCueMenuPopup::aboutToHide, this, &WOverview::slotCueMenuPopupAboutToHide);
+
+    m_pPassthroughLabel = new QLabel(this);
+    m_pPassthroughLabel->setObjectName("PassthroughLabel");
+    m_pPassthroughLabel->setAlignment(Qt::AlignLeft | Qt::AlignVCenter);
+    // Shown on the overview waveform when vinyl passthrough is enabled
+    m_pPassthroughLabel->setText(tr("Passthrough"));
+    m_pPassthroughLabel->hide();
+    QVBoxLayout *pPassthroughLayout = new QVBoxLayout(this);
+    passThroughLayout->setContentsMargins(0,0,0,0);
```

Comment:
You only changed the first occurrence.

```suggestion
    pPassthroughLayout->setContentsMargins(0,0,0,0);
```
and below

Reply:
Please fixup the broken commits in this branch and force-push to allow us to git bisect later.

Reply:
Thanks for checking! Done.